frontend/packages/dev-console/src/components/health-checks/RequestTypeForms.tsx

```diff
@@ -104,6 +105,7 @@ export const CommandRequestTypeForm: React.FC<RequestTypeFormProps> = ({ probeTy
     <TextColumnField
       name={`healthChecks.${probeType}.data.exec.command`}
       label="Command"
+      addLabel="Add command"
```

Comment:
should we go with **the first letter of every word** in caps or **the first letter of the text**  in caps? better to follow a single pattern

Reply:
Again, same thing - just a revert of what it was before you [removed it](https://github.com/openshift/console/pull/5998/files#diff-4349b9c69e6160a418fa47e212c28731L108).